test/lib/time.rb

```diff
@@ -50,14 +64,14 @@ def join_fragment(minutes_array)
   def create_new_fragments(result_array, input)
     first_minute = 0
     last_minute = 1439
-    result_array << [@sort_middle[0][1], last_minute] if input.length <= 1
+    result_array << [@sort_middle[0][1], last_minute] if @sort_middle.length <= 1
     if_more_two_fragments(result_array, input, last_minute)
     result_array << [first_minute, @sort_middle[0][0]]
     result_array.sort!
   end
 
-  def if_more_two_fragments(result_array, input, last_minute)
-    return unless input.length >= 2
+  def if_more_two_fragments(result_array, _input, last_minute)
```

Comment:
Why _input better then input?

Reply:
This variable doesn't need here. I forgot to delete it.